requirements/app.in

```diff
@@ -24,9 +24,10 @@ click
 email_validator
 rq
 rq-dashboard
-# the following two use environment markers (see PEP 496)
+# the following uses environment markers (see PEP 496)
 rq-win; os_name == 'nt' or os_name == 'win'
-redis; os_name == 'nt' or os_name == 'win'
+# This limit resolves a conflict with test.in. The culprit is fakeredis (check their setup.cfg)
+redis < 4.2.0
```

Comment:
Can you comment on removing the OS conditions?

Reply:
I believe my thought process was: We already have them in rq-win, and redis we need in any case.